.golangci.yml

```diff
@@ -135,3 +135,5 @@ issues:
         - goconst
         - gocyclo
         - gomnd
+        - dupl
+        - lll
```

Comment:
suggesting to add these to be excluded from test files. would want to apply to all repos if accepted.